origin/blobclient/cluster_client.go

```diff
@@ -108,6 +110,15 @@ func (c *clusterClient) defaultPollBackOff() backoff.BackOff {
 	}
 }
 
+func (c *clusterClient) CheckReadiness() error {
+	clients, err := c.resolver.Resolve(backend.ReadinessCheckDigest)
+	if err != nil {
+		return fmt.Errorf("resolve clients: %s", err)
+	}
+	shuffle(clients)
```

Comment:
Instead of shuffle can we check the readiness for a client at a random index between 0 to len(clients) ?

```golang
  rand.Seed(time.Now().UnixNano())
  randomIndex := rand.Intn(len(clients))
  return clients[randomIndex].CheckReadiness()
```
This avoid an overhead of shuffling clients in memory

Reply:
I'll make the change but there's no need to call rand.Seed(). According to [the docs](https://pkg.go.dev/math/rand#Seed), rand.Seed is already called at application startup.